Angular/src/app/pages/skill-quiz-generator/generated-quiz/generated-quiz.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input, SimpleChanges } from '@angular/core';
+import { FormBuilder, FormGroup, ReactiveFormsModule, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-generated-quiz',
+  standalone: true,
+  imports: [ReactiveFormsModule],
+  templateUrl: './generated-quiz.component.html',
+  styleUrl: './generated-quiz.component.scss'
+})
+export class GeneratedQuizComponent {
+  @Input() quiz: any = {};
+  quizForm: any = FormGroup;
```

Comment:
We need to promote best practises. `any` is not recommended. We should use a proper type.

Reply:
done